docs/maps/maps-aggregations.asciidoc

```diff
@@ -40,6 +40,18 @@ This field will be used in the terms aggregation to group your documents into en
 This field will be used to sort your documents in the top hits aggregation.
 . Set *Documents per entity* to configure the maximum number of documents accumulated per entity.
 
+[role="xpack"]
+[[point-to-point]]
+=== Point to point
```

Comment:
Is Point to point correct?  Or is it Point-to-point or Point-to-point connection?